osgp/shared/shared/src/test/java/org/opensmartgridplatform/shared/security/RsaEncryptionServiceTest.java

```diff
@@ -80,20 +95,24 @@ private void assertDecryptionOfKey(final RsaEncryptionService rsaEncryptionServi
         final byte[] expected = Files.readAllBytes(Paths.get(keyPath));
         final byte[] actual = rsaEncryptionService.decrypt(Hex.decodeHex(encryptedHexString.toCharArray()));
         assertThat(Hex.encodeHexString(actual)).withFailMessage("decrypted " + keyType)
-                .isEqualTo(Hex.encodeHexString(expected));
+                                               .isEqualTo(Hex.encodeHexString(expected));
     }
 
     @Test
     public void testEncryptDecryptReturnsInput() {
-        final KeyPair freshKeyPair = RsaEncryptionService.createKeyPair(KEYSIZE);
-        final RsaEncryptionService rsaEncryptionService = new RsaEncryptionService(freshKeyPair);
-        final byte[] input = this.createRandomInput();
-        final byte[] encrypted = rsaEncryptionService.encrypt(input);
-        final byte[] decrypted = rsaEncryptionService.decrypt(encrypted);
-        assertThat(decrypted).withFailMessage("decrypted bytes after encryption").isEqualTo(input);
+        try {
+            final KeyPair freshKeyPair = RsaEncryptionService.createKeyPair(KEYSIZE);
+            final RsaEncryptionService rsaEncryptionService = new RsaEncryptionService(freshKeyPair);
+            final byte[] input = this.createRandomInput();
+            final byte[] encrypted = rsaEncryptionService.encrypt(input);
+            final byte[] decrypted = rsaEncryptionService.decrypt(encrypted);
+            assertThat(decrypted).withFailMessage("decrypted bytes after encryption").isEqualTo(input);
+        } catch (EncrypterException ee) {
+            fail("Encryption failure", ee);
+        }
```

Comment:
It would be simpler to leave out the try catch block. If the exception occurs the test doesn't succeed, and this doesn't seem to offer much useful for the added code.